<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::WidgetInspector</class>
 <widget class="QWidget" name="GammaRay::WidgetInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="KFilterProxySearchLine" name="widgetSearchLine"/>
        </item>
        <item>
         <widget class="QTreeView" name="widgetTreeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="GammaRay::WidgetPreviewWidget" name="widgetPreviewWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="GammaRay::PropertyWidget" name="widgetPropertyWidget" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kde/kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header>propertywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::WidgetPreviewWidget</class>
   <extends>QWidget</extends>
   <header>tools/widgetinspector/widgetpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
